liberty-managed/JakartaEE9_README.md

```diff
@@ -1,12 +1,12 @@
-# Arquillian Liberty Managed with Jakarta EE 9 and 10
+# Arquillian Liberty Managed with Jakarta EE 9, 10 and 11
```

Comment:
Should we add something to this README about dropping support for Java 8 as of 3.x version, and if they need to run with Java 8 they should use a 2.x version?

Reply:
Something added